### DISCUSSION TITLE
/zh-cn/blog/posts/2024_summary

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#152] [Announcements]
# 2024 年度小结 - Reverier's Blog

日子过得真快，又到了写年度小结的时候了。
我以前似乎是没有写总结的习惯，可能是从这个博客开始，也可能是更早，每年都得写点东西煽情一下，不仅仅是为了记录，同样也是为了下一年能够继续在自己选择的道路上接着走下去。

https://blog.woooo.tech/posts/2024_summary/

**Comment by LittFlower** [comment]
前排支持！
